metta/match/ImplicationDirectIntroductionMatchTest.metta

```diff
@@ -1,4 +1,9 @@
 ;; Test Implication Direct Introduction Match Rule
 !(import! &self ImplicationDirectIntroductionMatch.metta)
 
-;; NEXT
+(≞ (P 42) (Bl True))
+(≞ (Q 42) (Bl False))
+(≞ (→ P Q) (ETV Empty (PrCnt 1 0)))
+(∉ 42 Empty)
+
+!(idi_induction &self)
```

Comment:
Files should end by a new line, it is explained here https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline